src/main/java/uk/gov/ons/census/action/builders/PrintFileDtoBuilder.java

```diff
@@ -9,6 +10,14 @@
 
 @Component
 public class PrintFileDtoBuilder {
+  private static final Set<ActionType> doesNotRequireUacQidActionTypes =
+      Set.of(
+          ActionType.P_RL_1RL1A,
+          ActionType.P_RL_1RL2BA,
+          ActionType.P_RL_2RL1A,
+          ActionType.P_RL_2RL2BA
+          // NO. DO NOT ADD ANYTHING TO THIS LIST. NOPE. NEVER.
```

Comment:
what does this comment actually mean? also isn't there more "reminder" action types than this like P_RL_2RL2BB? https://collaborate2.ons.gov.uk/confluence/display/SDC/RMC-293+-+Print+reminder+letters+and+questionnaires